src/ucp/wireup/wireup_cm.c

```diff
@@ -301,6 +302,11 @@ static unsigned ucp_cm_client_connect_progress(void *arg)
         goto out_unblock;
     }
 
+    status = uct_cm_client_ep_conn_notify(uct_cm_ep);
+    if (status != UCS_OK) {
+        ucp_ep_cleanup_lanes(ucp_ep);
```

Comment:
why it's needed? 
should not `ucp_worker_set_ep_failed` below handle the error?